packages/genesys-spark-components/src/components/stable/gux-listbox/option-group/gux-option-group.tsx

```diff
@@ -0,0 +1,54 @@
+import { Component, h, Host, JSX, Prop, State, Element } from '@stencil/core';
+import { hasSlot } from '@utils/dom/has-slot';
+
+/**
+ * The listbox component provides keyboard bindings and a11y patterns for selecting
+ * from a list of options.
+ *
+ * @slot - collection of elements conforming to the ListboxOptionElement interface
+ */
+@Component({
+  styleUrl: 'gux-option-group.scss',
+  tag: 'gux-option-group',
+  shadow: true
+})
+export class GuxOptionGroup {
+  @Element()
+  root: HTMLElement;
+
+  @Prop()
+  divider: boolean = true;
```

Comment:
What are the use cases for excluding the divider? It seems odd to me that if it is part of the Design System it is optional.

Reply:
The request was that the divider not be shown if the group is the last item in a listbox.